lib/template/function.h

```diff
@@ -29,18 +29,14 @@
 #include "plugin-types.h"
 #include "common-template-typedefs.h"
 
+#define TEMPLATE_INVOKE_MAX_ARGS 64
```

Comment:
Is it okay to limit the maximum arguments ? How did you come up with this number ?

Reply:
I consider it ok :) I am yet to see a config where we've used more than 15-20, that being the ewmm template function. There I'd even want to decrease the number by adding a couple of features to value-pairs.

If this is deemed too small, it's pretty easy to increment it, but by having this limit the code is much simpler. It is also an option to make it dynamic, but that's substantially more complex code.

I think it should be good for now.

Reply:
Okay, but because of this I mark it as user visible feature, we should at least document this limitation. (I'll notify the doc team.)